cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaConfigurationDiff.java

```diff
@@ -300,28 +289,25 @@ private Collection<AlterConfigOp> diff(NodeRef nodeRef, String desired,
                     .filter(configEntry -> configEntry.name().equals(pathValueWithoutSlash))
                     .findFirst();
 
+            boolean logConfigDiff = false;
```

Comment:
I can understand you are using this naming because in the end what you want is "should I log or not?"
But AFAIK, the meaning of the boolean returned by `removeProperty` and  `updateOrAdd` is that a configuration was actually changed (via a `DELETE` or `SET`) operation. It then translate into "let's log only if a configuration was changed". Should we find a better name for this varialble? i.e. `isConfigChanged`, `isConfigDiff` or whatever which is not strictly related to logging?